noxfile.py

```diff
@@ -149,3 +149,23 @@ def docs(session):
     session.cd("docs")
     session.run("make", "clean", external=True)
     session.run("make", "html", external=True)
+    
+
+@nox.session(python=["3.8", "3.9", "3.10"])
```

Comment:
We don't need multiple python session here. For integration this is important to make sure our package is working with multiple python versions. But for our tracking task, we can use just one python version.